lib/elixir/lib/code.ex

```diff
@@ -455,12 +455,21 @@ defmodule Code do
       If you set it to `false` later on, `do`-`end` blocks won't be
       converted back to keywords.
 
+    * `:normalize_bitstring_modifiers` (since v1.14.0) - when `true`,
+      removes un-necessary parentheses in known bitstring
+      [modifiers](`Kernel.<<>>/1`),
+      e.g. `<<foo::binary()>>` becomes `<<foo::binary>>`, or adds
+      parentheses for custom modifiers, e.g. `<<foo::custom_type>>`
+      becomes `<<foo::custom_type()>>`.
+      Defaults to `true`. This option changes the AST.
```

Comment:
We typically avoid `e.g.` and other abbreviations in Elixir for clarity!

```suggestion
    * `:normalize_bitstring_modifiers` (since v1.14.0) - when `true`,
      removes unnecessary parentheses in known bitstring
      [modifiers](`Kernel.<<>>/1`), for example `<<foo::binary()>>`
      becomes `<<foo::binary>>`, or adds parentheses for custom
      modifiers, where `<<foo::custom_type>>` becomes `<<foo::custom_type()>>`.
      Defaults to `true`. This option changes the AST.
```